### PR TITLE
Feat/qa landing update

### PR DESCRIPTION
**Pull request by JackMidd-Amido**
Feat: Extended coverage on testing overview and added contract testing documentation
https://amido-dev.visualstudio.com/Amido-Stacks/_workitems/edit/8409
https://amido-dev.visualstudio.com/Amido-Stacks/_workitems/edit/8412

#### 📲 What

Updated testing overview to include details of all testing available within Stacks.
Added a page for Provider Contract Testing in .Net Stack

#### 🤔 Why
		
To ensure documentation contains the latest and most relevant information
		
#### 🛠 How
		
Updating docs.

#### 👀 Evidence

see docs
		 
#### 🕵️ How to test

npm install
npm run start

#### ✅ Acceptance criteria Checklist

- [ ] Code peer reviewed?
- [ ] Documentation has been updated to reflect the changes?
- [ ] Passing all automated tests, including a successful deployment?
- [ ] Passing any exploratory testing?
- [ ] Rebased/merged with latest changes from development and re-tested?
- [ ] Meeting the Coding Standards?

### CODE REVIEW

File: docs/workloads/azure/backend/netcore/testing/provider_contract_testing.md [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+The following diagram shows an example flow of a Bi-Directional contract testing implementation with 1 provider and 1 consumer.
+
+![Bi-Directional Contract Testing](../../../../../../static/img/StacksQA_Bi-Directional-Contract-Testing.png)
+
+
+### Getting Started
+When creating a project using the [WebAPI template](../\quickstart\web_api\create_project_netcore.md), the `ci.yml` pipeline will include the steps required for Contract Testing as a provider.
+
+Within the contract testing steps, there are a number of variables that are required. In order for the pipeline to work as expected, the variable values will need to be updated before executing the pipeline.
+Variables can be found within `./build/azDevOps/azure/ci-vars.yml`
```
Comment (ninjagirl888) [inline docs/workloads/azure/backend/netcore/testing/provider_contract_testing.md:47]: I can't confirm the paths to say this bit as I can't see that file in the stacks-dotnet. has it yet to be merged in.  Also I notice that each offering has its own vars.yml so maybe need to update the file name.
Variables can be found within `./build/azDevOps/azure/ci-vars.yml`

image

**Review by ninjagirl888 — CHANGES_REQUESTED** [review]

File: docs/workloads/azure/backend/netcore/testing/provider_contract_testing.md (hunk 1 above)
Comment (JackMidd-Amido) [reply docs/workloads/azure/backend/netcore/testing/provider_contract_testing.md:47]: Yeah, I am working on a branch adding this.
I have created a project off of the .Net template and the vars are in that location

image

File: asciidoc/testing/testing_overview.adoc [hunk 2]
```diff
@@ -8,15 +8,48 @@ keywords: testing, QA, Ensono Stacks, automation, performance, accessibility, vi
...
-This summary outlines key areas of testing included in the Ensono Stacks.
+This summary outlines key areas of testing integrated within Ensono Stacks as well as some standalone solutions for testing Stacks applications.
+
+=== Unit Testing
+All of our Stacks come with Unit testing built in. This ensures that each individual unit (function/method) works in isolation.
+Examples:
+- link:https://github.com/Ensono/stacks-java/blob/master/java/src/test/java/com/amido/stacks/workloads/menu/api/v1/CategoryControllerTest.java[Java stack]
+- link:https://github.com/Ensono/stacks-dotnet/tree/master/src/cqrs/src/api/xxENSONOxx.xxSTACKSxx.API.UnitTests[DotNet stack]
+- link:https://github.com/Ensono/stacks-data/tree/main/tests/unit[Data stack]
+- link:https://github.com/Ensono/stacks-nx-plugins/blob/main/e2e/rest-client-e2e/tests/rest-client.spec.ts[NX plugins]
```
[the remark this answers is not in the document]
Comment (ninjagirl888) [reply asciidoc/testing/testing_overview.adoc]: Sorry the correct url is https://github.com/Ensono/stacks-nx-plugins/blob/main/packages/common/test/src/lib/stacks-attributes.spec.ts

File: docs/testing/testing_overview.md [hunk 3]
```diff
@@ -16,11 +16,43 @@ keywords:
...
-This summary outlines key areas of testing included in the Ensono Stacks.
+This summary outlines key areas of testing integrated within Ensono Stacks as well as some standalone solutions for testing Stacks applications.
+
+### Unit Testing
+All of our Stacks come with Unit testing built in. This ensures that each individual unit (function/method) works in isolation.
+Examples:
+- [Java stack](https://github.com/Ensono/stacks-java/blob/master/java/src/test/java/com/amido/stacks/workloads/menu/api/v1/CategoryControllerTest.java)
+- [DotNet stack](https://github.com/Ensono/stacks-dotnet/tree/master/src/cqrs/src/api/xxENSONOxx.xxSTACKSxx.API.UnitTests)
+- [Data stack](https://github.com/Ensono/stacks-data/tree/main/tests/unit)
+- [NX plugins](https://github.com/Ensono/stacks-nx-plugins/blob/main/e2e/rest-client-e2e/tests/rest-client.spec.ts)
```
[the remark this answers is not in the document]
Comment (ninjagirl888) [reply docs/testing/testing_overview.md]: Sorry the correct url is https://github.com/Ensono/stacks-nx-plugins/blob/main/packages/common/test/src/lib/stacks-attributes.spec.ts

**Review by ninjagirl888 — APPROVED** [review]

**Pull request merged**